Stormshield/stormshield_module/quarantined_file_actions.py

```diff
@@ -0,0 +1,15 @@
+from stormshield_module.base import StormshieldAction
+
+base_url = "/agents/{id}/tasks"
+
+
+class QuarantineFileAction(StormshieldAction):
+    verb = "post"
+    endpoint = base_url + "/file-quarantine"
+    query_parameters: list[str] = ["filePath", "bypassExcludedDirectories"]
+
+
+class RestoreQuarantinedFileAction(StormshieldAction):
+    verb = "post"
+    endpoint = base_url + "/quarantined-file-restoration"
+    query_parameters: list[str] = ["fileHashSha256", "overwriteExistingFile"]
```

Comment:
These fields must be passed in the request body; not as query parameters
```suggestion
    query_parameters: list[str] = []
```